ops/ansible/roles/bfd-server-load/test/ansible.cfg

```diff
@@ -0,0 +1,17 @@
+[defaults]
+# Ensure that we can connect without being prompted to verify host keys.
+# Reference: http://docs.ansible.com/ansible/latest/intro_getting_started.html#host-key-checking
+host_key_checking = False
+
+# Docker's AUFS and OverlayFS storage drivers don't support setfacl. This
+# should only be used in tests; **DO NOT** include this setting in production
+# environments.
+allow_world_readable_tmpfiles = True
+
+# More human readable errors when they happen
+stdout_callback = yaml
+
+# Provides summary profiling information
+# NOTE: callback_whitelist is deprecated in ansible >2.9 in favor of callbacks_enabled
+# callbacks_enabled = profile_tasks
+callback_whitelist = profile_tasks
```

Comment:
Given the above comment, should this be using callbacks instead?

Reply:
It might? Frankly, everything under `bfd-server-load/test/` was copied from the equivalent `test/` directory from the `bfd-db-migrator` Ansible role which also copied from the other Ansible roles within the `ops/ansible/roles/` directory. This appears to have just never been updated along the way, and as far as I can tell has not caused any problems.

@mjburling Should we be concerned with changing this, or just leave it be for now and save it for a future PR?

Reply:
Good question! In short, no. `callback_whitelist` is correct for ansible 2.9.

The configuration here was _largely_ copied from the other roles, where we _eagerly_ added it to assist whoever would be asked to pursue ansible upgrades. We haven't gotten there yet, and we're **still** on ansible 2.9.